hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestBulkLoadReplication.java

```diff
@@ -232,6 +253,23 @@ public void testBulkLoadReplicationActiveActive() throws Exception {
     assertEquals(9, BULK_LOADS_COUNT.get());
   }
 
+  @Test
+  public void testPartionedMOBCompactionBulkLoadDoesntReplicate() throws Exception {
+    Path path = createMobFiles(UTIL3);
+    ColumnFamilyDescriptor descriptor =
+      new ColumnFamilyDescriptorBuilder.ModifyableColumnFamilyDescriptor(famName);
+    PartitionedMobCompactor compactor = new PartitionedMobCompactor(UTIL3.getConfiguration(),
+      UTIL3.getTestFileSystem(), tableName, descriptor, Executors.newFixedThreadPool(1));
```

Comment:
Close the ThreadPool? I forget how we manage surefire forks, if this would orphan a thread for some reasonable length of time.

Reply:
Done.